src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/ReverseRunFunnelByConstant.java

```diff
@@ -0,0 +1,10 @@
+package edu.greenblitz.robotName.commands.shooter.funnel;
+
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.FunnelConstants;
+
+public class ReverseRunFunnelByConstant extends RunFunnelByPower {
```

Comment:
if @tnrz135 agrees consider making the command with 2 constructors: one with an input of thepower i.e. "new ReverseRunFunnel(0.5)" and one empty constructor that calls the other constructor with the default value